cmd/platform/export.go

```diff
@@ -27,66 +27,225 @@ func NewExportCommand() *cobra.Command {
+	cmd.Flags().Var(&pingoneRegion, profiles.PingOneRegionOption.CobraParamName, fmt.Sprintf("The region of the PingOne service(s). Allowed: %s.  Also configurable via environment variable %s", strings.Join(customtypes.PingOneRegionValidValues(), ", "), profiles.PingOneRegionOption.EnvVar))
 	profiles.AddFlagBinding(profiles.Binding{
-		Option: profiles.RegionOption,
-		Flag:   cmd.Flags().Lookup(profiles.RegionOption.CobraParamName),
+		Option: profiles.PingOneRegionOption,
+		Flag:   cmd.Flags().Lookup(profiles.PingOneRegionOption.CobraParamName),
 	})
-	profiles.AddEnvVarBinding(profiles.RegionOption)
+	profiles.AddEnvVarBinding(profiles.PingOneRegionOption)
+
+	cmd.MarkFlagsRequiredTogether(profiles.PingOneWorkerEnvironmentIDOption.CobraParamName, profiles.PingOneWorkerClientIDOption.CobraParamName, profiles.PingOneWorkerClientSecretOption.CobraParamName, profiles.PingOneRegionOption.CobraParamName)
+}
 
-	cmd.MarkFlagsRequiredTogether(profiles.WorkerEnvironmentIDOption.CobraParamName, profiles.WorkerClientIDOption.CobraParamName, profiles.WorkerClientSecretOption.CobraParamName, profiles.RegionOption.CobraParamName)
+func initPingFederateGeneralFlags(cmd *cobra.Command) {
+	// HTTPS host flag
+	cmd.Flags().String(profiles.PingFederateHttpsHostOption.CobraParamName, "", fmt.Sprintf("The PingFederate HTTPS host used to communicate with PingFederate's API.  Also configurable via environment variable %s", profiles.PingFederateHttpsHostOption.EnvVar))
+	profiles.AddFlagBinding(profiles.Binding{
+		Option: profiles.PingFederateHttpsHostOption,
+		Flag:   cmd.Flags().Lookup(profiles.PingFederateHttpsHostOption.CobraParamName),
+	})
+	profiles.AddEnvVarBinding(profiles.PingFederateHttpsHostOption)
 
-	return cmd
+	// Admin API path flag
+	cmd.Flags().String(profiles.PingFederateAdminApiPathOption.CobraParamName, "/pf-admin-api/v1", fmt.Sprintf("The PingFederate API URL path used to communicate with PingFederate's API.  Also configurable via environment variable %s", profiles.PingFederateAdminApiPathOption.EnvVar))
+	profiles.AddFlagBinding(profiles.Binding{
+		Option: profiles.PingFederateAdminApiPathOption,
+		Flag:   cmd.Flags().Lookup(profiles.PingFederateAdminApiPathOption.CobraParamName),
+	})
+	profiles.AddEnvVarBinding(profiles.PingFederateAdminApiPathOption)
+
+	// Require both HTTPS host and admin API path flags to be used together
+	cmd.MarkFlagsRequiredTogether(profiles.PingFederateHttpsHostOption.CobraParamName, profiles.PingFederateAdminApiPathOption.CobraParamName)
+
+	// X-Bypass-External-Validation header flag
+	cmd.Flags().Bool(profiles.PingFederateXBypassExternalValidationHeaderOption.CobraParamName, false, fmt.Sprintf("Header value in request for PingFederate. The connection test will be bypassed when set to true.  Also configurable via environment variable %s", profiles.PingFederateXBypassExternalValidationHeaderOption.EnvVar))
```

Comment:
"The connection test" sounds a little unclear, might be good to clarify that it's PF's own connection tests that get skipped, not something that pingctl is testing

Reply:
Updated